managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "authors"
+
+    def create(self, authors: str):
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (format) VALUES (?)",
+            (format_,)
+        )
+        self._connection.commit()
+
+    def all(self):
+        authors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        #print(f"authors_cursor: {authors_cursor}")
+        return [
+            Actor(*row) for row in authors_cursor
+        ]
+
+    def update(self, id_to_update: int, new_format: str):
```

Comment:
The `update` method should accept parameters for both `first_name` and `last_name` to update the actor's details. Adjust the method signature and SQL statement accordingly.